tsl/src/bgw_policy/job.c

```diff
@@ -185,6 +208,27 @@ policy_reorder_execute(int32 job_id, Jsonb *config)
 	return true;
 }
 
+void
+policy_reorder_read_config(Jsonb *config, Hypertable **hypertable, Oid *index_relid)
```

Comment:
I do not think there is any "base class" to create here so not sure about that part, but otherwise was something that I considered. I will add the shared structure if there is a reason for it. Otherwise just stick to one for each since the dynamic nature is not needed.

Reply:
W.r.t. base class, it is not only for having common fields, but also for typing, e.g., ability to have function interfaces that return the base type as opposed to different interfaces for each policy.